src/bun.js/bindings/webcore/JSWebSocket.cpp

```diff
@@ -230,10 +230,25 @@ static inline JSC::EncodedJSValue constructJSWebSocket3(JSGlobalObject* lexicalG
                 RETURN_IF_EXCEPTION(throwScope, encodedJSValue());
             }
         }
+
+        if (JSValue tlsOptionsValue = options->getIfPropertyExists(globalObject, PropertyName(Identifier::fromString(vm, "tls"_s)))) {
+            if (!tlsOptionsValue.isUndefinedOrNull() && tlsOptionsValue.isObject()) {
+                if (JSC::JSObject* tlsOptions = tlsOptionsValue.getObject()) {
+
+                    if (JSValue rejectUnauthorizedValue = tlsOptions->getIfPropertyExists(globalObject, PropertyName(Identifier::fromString(vm, "rejectUnauthorized"_s)))) {
```

Comment:
we _technically_ need to check for exceptions here incase the developer does something insane like

```js
new WebSocket(url, {tls: { get rejectUnauthorized() { throw Symbol("123"); } })
```